CHANGELOG.md

```diff
@@ -83,6 +83,8 @@
 
 - **Zipkin**: Fix an issue where the global plugin's sample ratio overrides route-specific.
   [#9877](https://github.com/Kong/kong/pull/9877)
+- **JWT**: Deny requests that have different tokens in the jwt token search locations. Thanks Jackson 'Che-Chun' Kuo from Latacora for reporting this issue.
+  [#9946](https://github.com/Kong/kong/pull/9946)
```

Comment:
Would it make sense to call this out as a "potential breaking change" for some users?

Reply:
In the scenario that carries both a JWT token in the query string and another non-JWT token in the JWT header search location in the same request, then yes, this is a breaking change.  In my opinion, this usage is quite strange.

Reply:
I agree that the usage is quite strange. I've encountered [Hyrum's Law](https://www.hyrumslaw.com/) too many times to assume that this won't be a problem for anyone at all. I hope that is the case. Documenting the change of behavior helps us defend the reason for this change and also educate users.

We should also call this change out in the plugin manual.